repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/LogFormatArgument.java

```diff
@@ -43,13 +43,13 @@
     REQUEST_PROTOCOL("H"),
     RESPONSE_HEADER("o"),
     STRING("STRING"),
-    ERROR_MESSAGE("M"),
+    RESPONSE_REASON("M"),
     RESPONSE_TIME_MICROSECONDS("D"),
     RESPONSE_TIME_SECONDS("T");
 
     private final String argument;
 
-    private LogFormatArgument(String argument) {
+    LogFormatArgument(String argument) {
```

Comment:
I don't believe this is actually a change in scope, just dropping a redundant scope declaration. This is one of the things IntelliJ highlighted.

From https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html:
```
Note: The constructor for an enum type must be package-private or private access. It automatically creates the constants that are defined at the beginning of the enum body. You cannot invoke an enum constructor yourself.
```